.changeset/use-consistent-type-definitions.md

```diff
@@ -0,0 +1,37 @@
+---
+"@biomejs/biome": minor
+---
+
+feat(lint): add `useConsistentTypeDefinitions` rule
```

Comment:
We don't use conventional commits style in changesets.

```suggestion
Added `useConsistentTypeDefinitions` rule.
```

Reply:
addressed